### PR TITLE
Fix for the html:link syntax

### PR DESCRIPTION
**Pull request by mfdewit**
Just a minor correction on Consumer method name and URI syntax.

### CODE REVIEW

File: _docs/flavour/component-packages/html.md [hunk 1]
```diff
@@ -231,14 +231,14 @@ Syntax:
...
-  This lambda takes `Comsumer<String>` as a single parameter and must call `apply()` method.
+  This lambda takes `Comsumer<String>` as a single parameter and must call `accept()` method.
   Such strange API is required for integration with routing library.
   If you want to generate target manually, you better use [attr:href](/docs/flavour/component-packages/attr.html).
 
 Example:
 
 ```html
-<a html:link="it.apply('#employees/' + employee.id)">
+<a html:link="it.accept('/employees/' + employee.id)">
```
Comment (konsoletyper) [inline _docs/flavour/component-packages/html.md:241]: What is the purpose of this change?

Comment (mfdewit) [reply _docs/flavour/component-packages/html.md:241]: I suppose you are referring to the URI syntax change? 
When in a routed application, the # symbol is appended to the already present #.
So, in my case this became: index.html##/employees.

That's the reason I proposed this change, since it is likely this link is displayed on a page already in a specific route. But I might be mistaken here :)
